Index.html

```diff
@@ -38,17 +38,209 @@ <h2>
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+                <div class="flex-box">
+                    <img src="./Images/SecondPortfolio.svg" alt="My-Work">
+                    <h1>Multi-Post Stories Gain+Glory</h1>
+
+                    <div class="list-box">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavaScript</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                  
+                    <div class="See-btn">     
+                        <button type="button" class="see-project">See Project</button>
+                    </div>
+
+                </div>
+            </div>
```

Comment:
We have a list of projects here. This looks like something we're familiar with :thinking: 
I think we can use the HTML list (`<ul>`) here. The list of projects should use a `ul` tag to group them, and `li` per project :blush: